documentation/catalog-content.md

```diff
@@ -30,6 +30,13 @@ Feel free to add suggestions for new slugs to the lists as part of your PR! Be s
 - 'paths/back-end-engineer-career-path'
 ```
 
+### Command Line
+
+```
+- "learn-the-command-line"
+- "paths/computer-science"
```

Comment:
```suggestion
- 'learn-the-command-line'
- 'paths/computer-science'
```

Reply:
@yangc95 thank you for the suggestion! I have updated the code.